ezua/whitelist_disks/README.md

```diff
@@ -0,0 +1,50 @@
+
+# Introduction to Whitelisting disks
+
+A user may have many disks on a node and does not want Data Fabric to use all of them.  A user can choose which disks are used by Data Fabric on each node.  This process is called whitelisting disks.  The disks that will be used on each node can be different. 
+
+The list of disks will be provided in a text file called /mnt/whitelist.  Here is a sample: 
+
+m2-lr1-dev-vm20.mip.storage.hpecorp.net: /dev/sdb  
+m2-lr1-dev-vm21.mip.storage.hpecorp.net: /dev/sdc  
+m2-lr1-dev-vm22.mip.storage.hpecorp.net: /dev/sdb,/dev/sdc  
+m2-lr1-dev-vm23.mip.storage.hpecorp.net: /dev/sdd
+
+The whitelist file can have FQDN or IP addresses.
+
+The file with the list of disks needs to be copied to each worker node before UA is installed.  The file must be called "whitelist" and must be placed in /mnt.  The user may need sudo privileges to place the whitelist file in /mnt.  The push_whitelist bash script is provided to push the whitelist file to each node specified in the file.  It is optional to use this script. The user can distribute the whitelist file themselves.
+
+# Prerequisites
+
+The bash script is designed to run on the kubernetes master node for the workload cluster.  It uses sshpass to copy the whitelist file to each worker node. Please install sshpass on the master node.
+
+The bash script will look for a file called whitelist in the local directory and push this file to /mnt on each worker node.  Verify that the file can be read by the user running the installer.
+
+# Usage
+
+The bash script will prompt for a username and password that can be used to log into each worker node.
+
+# Sample output
+
+./push_whitelist  
+Enter Username :   
+`<username>`  
+Enter Password :***************  
```

Comment:
we should move these lines above the #27

Reply:
fixed

Reply:
also, we should say, this script should be executed prior to installation. Why do we say, it designed to run on the k8s master node? is our goal is to copy this file on every node before we start the installation right?

Reply:
Yes, I have these two lines:

The whitelist file needs to be copied to each worker node before UA is installed.  It should be copied to each worker before the UI is started.  

I did not mention the script because it is optional.  Do you want me to assume they will use the script?

I mentioned the k8s master node to make it "simpler".  But I'll take it out.  Any node that has access to the worker nodes should work.

Yes, our goal is to copy this file on every node before we start the installation.

Reply:
Running the push_whitelist bash script on a mac is a little painful.  I don't believe mac allows sshpass.  I think you have to type in your password for each node.

Reply:
I re-wrote the comments to assume they will use the push_whitelist script.  They can decide themselves if they want to push the file in another way (i.e. clush)